### ISSUE TITLE
Investigate running CI jobs in parallel where possible.

### ISSUE THREAD

**Issue by bear-in-the-air** [daft-engineers/vocoder#49]
As a daft engineer I don't want to wait long for my CI tasks to finish

Success criteria:

- If possible, CI jobs should run in parallel

**Comment by MHCooke** [comment]
Self hosted runners cannot run jobs in parallel. The best we can do is register a second runner, which means that if two merge requests go up at the same time they do not need to queue side by side.

**Comment by MHCooke** [comment]
Because jobs are stateful, they cannot be deployed across multiple runners. Will raise a new low priority issue to investigate making CI jobs stateless.
